base/linalg/exceptions.jl

```diff
@@ -32,6 +32,15 @@ end
 struct PosDefException <: Exception
     info::BlasInt
 end
+function Base.showerror(io::IO, ex::PosDefException)
+    print(io, "PosDefException: matrix is not ")
```

Comment:
I think `PosDef` is fine. We use the definition that requires the matrix to be `Hermitian` consistently. Some users have been taught the other definition that allows for non-Hermitian matrices but I think the error message is sufficiently clear about the reason for the failure.